x-pack/plugins/security_solution/public/flyout/right/hooks/use_assistant.test.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RenderHookResult } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react-hooks';
+import type { UseAssistantParams, UseAssistantResult } from './use_assistant';
+import { useAssistant } from './use_assistant';
+import { mockDataFormattedForFieldBrowser } from '../mocks/mock_context';
+import { useIsExperimentalFeatureEnabled } from '../../../common/hooks/use_experimental_features';
+import { useAssistantOverlay } from '@kbn/elastic-assistant';
+
+jest.mock('../../../common/hooks/use_experimental_features');
+jest.mock('@kbn/elastic-assistant');
+
+const dataFormattedForFieldBrowser = mockDataFormattedForFieldBrowser;
+const isAlert = true;
+
+const renderUseAssistant = () =>
+  renderHook((props: UseAssistantParams) => useAssistant(props), {
+    initialProps: { dataFormattedForFieldBrowser, isAlert },
+    // wrapper: ProviderComponent,
```

Comment:
remove commented line?

Reply:
oops, left over from something I was testing before, fixed thank you!